configs/components/ruby-2.4.2.rb

```diff
@@ -207,6 +216,9 @@
     end
   end
 
+  # TODO: Remove this once PA-1607 is resolved.
+  pkg.configure { [ "/opt/pl-build-tools/bin/autoconf" ] } if platform.is_aix?
+
```

Comment:
It looks like Travis is failing in a couple of spots with a rubocop [style error](https://travis-ci.org/puppetlabs/puppet-agent/jobs/284379356):
 
`Style/SpaceInsideBrackets: Space inside square brackets detected.`